### DISCUSSION TITLE
[Dynamic Link] I want to get a dynamic link to my app from the flutter web. Is this possible?

### DISCUSSION THREAD

**Discussion by myeongseop-sofit** [firebase/flutterfire#10405] [Q&A]
I want to get a dynamic link to my app from the flutter web. 
Is this possible?

**Comment by rusudinu** [comment]
You could do that by using a post request to Dynamiclinks API, check [this](https://firebase.google.com/docs/dynamic-links/rest) out. The dynamic links flutter plugin is not yet supported for web, that's why you have to manually call the API. You can also check out [this](https://stackoverflow.com/a/69458558/8647107) for a code snippet.
